src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -3,18 +3,129 @@
 import React from "react";
 import { Meta, Story } from "@storybook/react";
 import { Tooltip as TooltipComponent, TooltipProps } from "./Tooltip";
+import { BrightHeaderStyle } from "./BrightHeader";
+import IconIcons from "@elements/Icon/Generated/IconIcons";
 
 export default {
     title: "Components/Tooltip",
     component: TooltipComponent,
     args: {
-        tooltip: "Tooltip Content (Reactnode)",
+        tooltip: "Cupcake ipsum dolor sit amet ice cream.",
+        heading: "",
+    },
+    argTypes: {
+        brightHeader: {
+            options: ["None", "Information", "Warning", "Tip", "Note"],
```

Comment:
Right now, it doesn't show which option is selected, this should fix it:
```suggestion
            options: ["None", ...Object.values(BrightHeaderStyle)],
```